### PR TITLE
Updated Mongolian carrier prefixes

### PR DESCRIPTION
**Pull request by digz6666**
Updated from Communications Regulatory Commission of Mongolian.
http://www.crc.gov.mn/k/4L/36

Another source: http://www.wtng.info/wtng-976-mn.html

### CODE REVIEW

File: resources/carrier/en/976.txt [hunk 1]
```diff
@@ -13,9 +13,16 @@
 # limitations under the License.
 
 97680|Unitel
-976830|G-Mobile
-976831|G-Mobile
+97683|G-Mobile
```
Comment (kkeshava) [inline resources/carrier/en/976.txt]: We could not find 83 complete range assigned to G-Mobile in the evidence you provided. Looks like only 830 and 831 are assigned https://www.itu.int/dms_pub/itu-t/oth/02/02/T020200008E0002PDFE.pdf.
Could you please revert this?

File: resources/carrier/en/976.txt [hunk 2]
```diff
@@ -13,9 +13,16 @@
...
 97685|Mobicom
 97686|Unitel
 97688|Unitel
-97689|Unitel
\ No newline at end of file
+97689|Unitel
```
Comment (kkeshava) [inline resources/carrier/en/976.txt]: We already had an entry for 97689|Unitel. Please revert this?

File: resources/carrier/en/976.txt (hunk 1 above)
Comment (digz6666) [reply resources/carrier/en/976.txt]: I think we can safely assume that all 83xx are assigned to G-Mobile.
Also this PDF document you provided and the link of the crc.gov.mn is outdated, it doesn't include Unitel 976-86 and Mobicom 976-85 which assigned to them at later date.

File: resources/carrier/en/976.txt (hunk 2 above)
Comment (digz6666) [reply resources/carrier/en/976.txt]: There's just 1 entry for 97689|Unitel, I don't know why github shows this as a change.

Comment (digz6666) [reply resources/carrier/en/976.txt]: If you could see the whole file :)

File: resources/carrier/en/976.txt (hunk 1 above)
Comment (digz6666) [reply resources/carrier/en/976.txt]: You can see that Mobicom is selling numbers started with 976-85 here as a reference: https://www.mobicom.mn/en/newnumber/mobicom

Comment (kkeshava) [reply resources/carrier/en/976.txt]: 976 86 is Unitel we have IR 21 doc for that. 976 85 is mentioned only in crc.gov.mn. And for 83xx which is assigned to G-Mobile, this is not mentioned even in the latest IR 21 doc so we can't make it generic. We need official evidence to do so.

File: resources/carrier/en/976.txt (hunk 2 above)
Comment (kkeshava) [reply resources/carrier/en/976.txt]: No problem, then we can ignore that :)

File: resources/carrier/en/976.txt (hunk 1 above)
Comment (digz6666) [reply resources/carrier/en/976.txt]: For G-Mobile they're transferring 976-53xx-xxx users into 976-83xx-xxxx starting from June of 2014 as they announced here, don't know why its not published on crc.gov.mn: https://www.g-mobile.mn/en/ns/51.html

Comment (digz6666) [reply resources/carrier/en/976.txt]: Seems like G-Mobile released 53xx numbers for mobile by violating regulations where 5xxx numbers are for wireless telephones, not mobiles, so they transferred their customers to 83xx, there's news here: http://unuudur.mn/article/39836

Comment (kkeshava) [reply resources/carrier/en/976.txt]: The document https://www.g-mobile.mn/en/ns/51.html is more than 3 years old, we are unable to translate it as it is in image format. We need evidence that we can understand. 
It seems like 53\d{6} used to be mobile, renumbered to 83\{6}, then 53\d{6} was made available to fixed-line, but we are not sure of it. Apart from that, even a document evidencing that a renumbering from 53\d{6} to 83\d{6} happened does not mean that 83\d{6} is completely assigned.

Comment (kkeshava) [reply resources/carrier/en/976.txt]: The doc doesn't fit our evidence guidelines. We request you to file a separate validation issue for 53\d}6} migrating to 83\d{6} range.
We will push this PR with other changes that you have made, so please revert 97683|G-Mobile

Comment (digz6666) [reply resources/carrier/en/976.txt]: @kkeshava I've reverted 97683|G-Mobile as you requested.

Comment (digz6666) [reply resources/carrier/en/976.txt]: The text on the image is here:

53xx xxxx дугаарлалт дээрх хэрэглэгчдийн ашиглаж буй үйлчилгээний нөхцөлийг өөрчлөлгүйгээр хэрэглэгчийн сонгосон дугаарлалт руу шилжүүлэх ажил 2014 оны 6-р сараас эхлэн үе шаттайгааар зохион байгуулагдсан. Тухайн хугацаанд шилжүүлээгүй дугааруудыг 09-р сарын 30-аас 10-р сарын 01-нд шилжих шөнө 83xx xxxx дугаарлалт руу сүүлийн 6 орон болон үйлчилгээний нөхцөл өөрчлөлгүйгээр шилжүүлэх болсныг мэдэгдье.

Comment (digz6666) [reply resources/carrier/en/976.txt]: I've asked from G-Mobile myself and they said they're selling numbers from 8300 xxxx to 8319 xxxx.

**Comment by kkeshava** [comment]
Thanks, could you please update your branch and merge it.

**Comment by digz6666** [comment]
@kkeshava I've committed your requested changes and it shows here. Any other things that missing?

**Comment by kkeshava** [comment]
Thanks :)

**Pull request merged**